care/emr/api/viewsets/user.py

```
@@ -192,3 +193,69 @@ def pnconfig(self, request, *args, **kwargs):
+        return Response(
+            {
+                "id": str(service_account.external_id),
+                "username": service_account.username,
+                "is_service_account": True,
+                "message": "Service account created successfully.",
+            },
+            status=201,
+        )
+
+    @action(detail=True, methods=["POST"])
+    def generate_service_account_token(self, request, *args, **kwargs):
```

Ruff (0.14.10)

Unused method argument: `args`

(ARG002)